package.json

```diff
@@ -14,7 +14,7 @@
     "build": "nx build winglang",
     "contributors:add": "all-contributors add",
     "contributors:check": "all-contributors check",
-    "contributors:generate": "all-contributors generate",
+    "contributors:update": "all-contributors check | grep \"Missing contributors\" -A 1 | tail -n1 | sed -e \"s/,//g\" | xargs -n1 | grep -v \"\\[bot\\]\" | xargs -n1 -I{} all-contributors add {} code",
```

Comment:
![image](https://user-images.githubusercontent.com/1237390/220403037-c5026143-c068-47f3-b5a0-b936fd7e4642.png)

Reply:
What was wrong with `generate`?

Reply:
😆 

I'm not sure but `generate` didn't seem to actually create / update the list for me